doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -112,26 +111,59 @@ Networks nodes are mainly used for L3 traffic management for overlay tenant netw
 | BIOS/boot Parameter | Value |
 |--------------------|--------------------|
 | Boot disks |RAID 1 |
-| <to be filled if needed>|  |
-| …|  
+  
  
 -	HW specifications
 -	How many nodes to meet SLA
 -	Sizing rules
 
 #### 4.2.2.5. Compute Nodes
+
+This section specifies the compute node configurations to support the flavors. Because a flavor has the same capabilities but different geometry sizes, the term “flavor series” is used.
+
 -	The software and hardware configurations are as specified in the [Reference Model chapter 5.4](../../../ref_model/chapters/chapter05.md#5.4)
 -	BIOS requirement
-    -	The general bios requirements are described in the [Reference Model chapter 5.4](../../../ref_model/chapters/chapter05.md#5.4)
-    -	Additionally, for OpenStack we need to set the following boot parameters:
+    -	The general BIOS requirements are described in the [Reference Model chapter 5.4](../../../ref_model/chapters/chapter05.md#5.4)
+
+Reference Model Chapter 4 [Table 4-17](../../../ref_model/chapters/chapter04.md#4211-predefined-compute-flavours) specifies flavor geometry and capabilities.  For convenience, the flavor geometry is reproduced in Table 4-1. 
+
+**Flavor Geometry**
+
+| .conf | vCPU (c) | RAM (r) | Local Disk (d) | Management interface |
+|----|----|----|----|----|
+| .tiny | 1 | 512 MB | 1 GB | 1 Gbps |
+| .small | 1 | 2 GB | 20 GB  | 1 Gbps |
+| .medium | 2 | 4 GB | 40 GB | 1 Gbps |
+| .large | 4 | 8 GB | 80 GB | 1 Gbps |
+| .2xlarge* | 8 | 16 GB | 160 GB | 1 Gbps |
+| .4xlarge* | 16 | 32 GB | 320 GB | 1 Gbps |
+| .8xlarge* | 32 | 64 GB | 640 GB | 1 Gbps |
+
+<p align="center"><b>Table 4-1: Flavor Geometries</b></p>
+
+**Flavor Series**
+
+The Reference Model specifies the Basic (B) and Network Intensive (N) instance types (or flavor series). The Reference Model also specifies support for two different CPU allocation ratio values for the Basic flavor types, and choice of network acceleration capabilities utilising DPDK and SR-IOV technologies; please note SR-IOV is supported under an exception policy. Table 4-2 lists the capabilities for each flavor series.
+
+| Flavor Series | CPU Allocation Ratio | SMT | CPU Pinning | NUMA | Huge Pages | Data Traffic |
+|----|----|----|----|----|----|----|
+| B1 | 1:1 | Y | N | N | N | OVS-kernel |
+| B4 | 4:1 | Y | N | N | N | OVS-kernel |
+| NV | 1:1 | Y | Y | Y | Y | OVS-kernel |
+| ND | 1:1 | Y | Y | Y | Y | OVS-DPDK |
+| NS | 1:1 | Y | Y | Y | Y | SR-IOV with OVS-kernel |
```

Comment:
where is OVS involved with SRIOV ?

Reply:
@deebhatia for OAM traffic only

Reply:
If I select NS, OVS will not be involved in data traffic (column title), right?

Reply:
@deebhatia Correct -- didn't look at the column header. Corrected. Thanks

Reply:
Okay